android/java/org/chromium/chrome/browser/BraveSyncReflectionUtils.java

```diff
@@ -21,26 +21,19 @@
     public static Object getSyncWorker() {
         // May be invoked in non-UI thread when we do validation for camera QR in callback
         if (!sInitialized) {
-            if (BraveConfig.SYNC_ENABLED) {
-                try {
-                    sBraveSyncWorker =
-                        Class.forName("org.chromium.chrome.browser.BraveSyncWorker")
-                             .getConstructor()
-                             .newInstance();
-                } catch (Exception e) {
-                    Log.e(TAG, "Cannot create BraveSyncWorker ", e);
-                }
+            try {
+                sBraveSyncWorker = Class.forName("org.chromium.chrome.browser.BraveSyncWorker")
```

Comment:
This whole file should be removed. If we now don't have `BraveConfig.SYNC_ENABLED` - we can always create `new BraveSyncWorker()` and invoke `BraveSyncInformers.show()` and don't need to create classes with reflection.

Places with call `BraveSyncReflectionUtils.showInformers()` should be changed to `BraveSyncInformers.show()`.

And `BraveSyncReflectionUtils.getSyncWorker()` should be moved to static `BraveSyncWorker.get()` and be replaced over the code.